_posts/2018-12-06-How-to-do-quick-revie-towards-tickets-and-bugs-before-release.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "How to do quick-review of tickets and bugs before release?"
+date:   2018-12-06 22:49:39 +0800
+
+---
+As we all know, before GA, there are tickets or bugs that we need to decide if they should be fixed or lower their priority or translate to Improvement.
+ This essay is for improving our efficiency toward review ticket.
+
+Before we discuss this topic, I want to share my understanding toward the definition of Bug, P3 bug and improvement.
+
+In my opinion, bug means current flow or operation will cause customer do mistake during work, which means our product not only lowers customer's efficiency, but also leads to customer's trouble.
+
+**The examples:**
+  * user can not dial out a phone number
+  * the task that user is editing is overwrote by us, or at call    
+  * At Call History or Active Call List, our product displays wrong phone numbers or wrong call direction/duration.
+
+
+So following this definition, we can have a better understanding toward P3 bug and improvement.
+As we know, P3 bug means it is a bug but not so important.
+
+BUT what does 'not so important' mean?
+It means it will cause user do mistake but the frequency of mistake is hardly.
```

Comment:
similarly, 'it will cause difficulties for users'.
You can say 'something hardly ever happens', but frequency is hardly is not correct. Just use 'frequency of mistakes is low'.